src/components/room-invitation-creation-modal.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import Logger from '../common/logger.js';
+import { Form, Modal, Input } from 'antd';
+import { useTranslation } from 'react-i18next';
+import errorHelper from '../ui/error-helper.js';
+import { useService } from './container-context.js';
+import React, { useState, useRef } from 'react';
+import RoomApiClient from '../api-clients/room-api-client.js';
+
+const FormItem = Form.Item;
+
+const logger = new Logger(import.meta.url);
+
+function RoomInvitationCreationModal({ isVisible, onClose, roomId }) {
+  const formRef = useRef(null);
+  const roomApiClient = useService(RoomApiClient);
+  const { t } = useTranslation('roomInvitationCreationModal');
+
+  const emailValidationRules = [
+    {
+      required: true,
+      message: t('emailRequired'),
+      whitespace: true
+    }
+  ];
+  const initialFormValues = { email: null };
+  const [loading, setIsLoading] = useState(false);
+
+  const handleFormFinish = async values => {
+    try {
+      setIsLoading(prevState => ({ ...prevState, loading: true }));
```

Comment:
Probably a leftover, in all the `setIsLoading` you can just say `setIsLoading(true/false)`

Reply:
Yes, thanks, I also renamed to isLoading so we have the pattern x setX